.github/workflows/awstest.yml

```diff
@@ -26,5 +28,10 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{secrets.AWS_KEY_ID}}
           AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_KEY_SECRET}}
           TOWER_ACCESS_TOKEN: ${{secrets.TOWER_ACCESS_TOKEN}}
-        run: | # Submits job to AWS batch using a 'nextflow-big' instance. Setting JVM options to "-XX:+UseG1GC" for more efficient garbage collection when staging remote files.
-          aws batch submit-job --region eu-west-1 --job-name nf-core-viralrecon --job-queue 'default-8b3836e0-5eda-11ea-96e5-0a2c3f6a2a32' --job-definition nextflow-4GiB --container-overrides '{"command": ["nf-core/viralrecon", "-r '"${GITHUB_SHA}"' -profile test_full --outdir s3://nf-core-awsmegatests/viralrecon/results-'"${GITHUB_SHA}"' -w s3://nf-core-awsmegatests/viralrecon/work-'"${GITHUB_SHA}"' -with-tower"], "environment": [{"name": "TOWER_ACCESS_TOKEN", "value": "'"$TOWER_ACCESS_TOKEN"'"}, {"name": "NXF_OPTS", "value": "-XX:+UseG1GC"}]}'
+        run: | # Submits job to AWS batch using a 'nextflow-4GiB' instance. Setting JVM options to "-XX:+UseG1GC" for more efficient garbage collection when staging remote files.
+          aws batch submit-job \
+          --region eu-west-1 \
+          --job-name nf-core-viralrecon \
+          --job-queue 'default-8b3836e0-5eda-11ea-96e5-0a2c3f6a2a32' \
```

Comment:
```suggestion
          --job-queue $AWS_JOB_QUEUE \
```

Reply:
Will be fixed in https://github.com/nf-core/viralrecon/pull/118